packages/ui-library/src/components/transaction-receipt/transaction-types-map.ts

```diff
@@ -0,0 +1,13 @@
+import { TransactionEventTypes } from 'common-types';
+import { defineMessage } from 'react-intl';
+
+export const TRANSACTION_TYPE_TITLE_MAP = {
```

Comment:
We could already implement native transfers, the indexer is already returning those.
- For naming the event:
https://github.com/Mean-Finance/mean-api/blob/dev/src/accounts/handlers/accounts-handler.ts#L170
- For the event interface:
https://github.com/Mean-Finance/mean-api/blob/4ec25a15052a017344def6ba4c55e281db38c824/src/shared/ports/db/repositories/factories/native-repository.ts#L50